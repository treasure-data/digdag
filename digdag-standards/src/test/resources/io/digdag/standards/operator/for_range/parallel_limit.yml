_command:
  from: 0
  to: 4
  slices: 4
_parallel:
  limit: 2
_do:
  echo>: do
