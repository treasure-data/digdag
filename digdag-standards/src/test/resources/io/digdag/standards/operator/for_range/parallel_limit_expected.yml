+range-from=0&to=1:
  echo>: do
  _export:
    range: {from: 0, to: 1, index: 0}
+range-from=1&to=2:
  echo>: do
  _export:
    range: {from: 1, to: 2, index: 1}
+range-from=2&to=3:
  echo>: do
  _export:
    range: {from: 2, to: 3, index: 2}
+range-from=3&to=4:
  echo>: do
  _export:
    range: {from: 3, to: 4, index: 3}
_parallel:
  limit: 2
